chutney/ui/src/app/shared/components/execute-modal/scenario-execute-modal.component.html

```diff
@@ -25,6 +24,21 @@
   </div>
   }
 
+  @if (jiraLink) {
```

Comment:
<img width="696" height="265" alt="image" src="https://github.com/user-attachments/assets/fd6555c6-6e16-40b4-8106-e449e0c21470" />

Please follow the cosmetics used in example (https://primeng.org/radiobutton) :
* Add `flex` classes to parent and `div` containers
* Add `for` and `class` to labels
* Add some margin top to the new jira id `input`

Also, if possible, it would be nice to add for the first two radio buttons the associated jira ids as links to the Jira backend used (with icon), to know which id will be used when re-execute.